sample_scripts/third_party_platform_exporting/magento/v1.9/soap_api/orders.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+require 'savon'
+require 'json'
+
+require_relative 'transporter_exporter'
+
+class OrderExporter < TransporterExporter
+  def run
+    orders_for(required_env_vars['MAGENTO_STORE_ID']).each do |order|
```

Comment:
I'd prefer if this was done similar to `CustomerExporter` where the filtering of the store is done within the `customers` method, and we use `customers` here. Perhaps change the current `orders` method to `all_orders`?

Reply:
Yeah--I like that, the `MAGENTO_STORE_ID` is required, so there shouldn't be more than a single method.